examples/cross_kms_support/README.md

```diff
@@ -0,0 +1,10 @@
+# Cross account KMS encryption example
+
+A simple example that shows how to provision a VPC with single subnet in a single zone with a public gateway enabled OCP VPC cluster with cross account KMS encryption.
+
```

Comment:
we need to add a about having the cross account auth policy set set before running the example